index.html

```diff
@@ -45,6 +45,15 @@ <h4 class="card-title">Truc Nguyen, Teacher</h4>
             <a href="https://github.com/trucnguyen" target="_blank"><i class="fa fa-github fa-2x" title="Github"></i></a>
           </div>
         </div>
+		
+		<div class="card">
```

Comment:
Nitpicky, but make sure the indentation is consistent (I believe we're using 2 space tabs)